cmd/mapkubeapis/map.go

```diff
@@ -36,7 +35,6 @@ type MapOptions struct {
 	MapFile          string
 	ReleaseName      string
 	ReleaseNamespace string
-	RunV2            bool
 	StorageType      string
 	TillerOutCluster bool
```

Comment:
Do you want to also remove the `StorageType` and `TillerOutCluster` fields here?

If so, they can probably also be removed in `common.go`.

Reply:
Sure, good catch. Removed.